### ISSUE TITLE
Using both cluster and type for vcovCL

### ISSUE THREAD

**Issue by lorenzoFabbri** [vincentarelbundock/marginaleffects#971]
I'd like to know whether it is possible to specify both `cluster`  and `type` for `vcovCL`:

```
avg_predictions(vcov = ~cluster, type = "HC3")
```

I see that `type` is referred for something else in `avg_predictions`. Thank you.

Reference: https://github.com/vincentarelbundock/marginaleffects/blob/2b9f2d97b964eeef9f90a0e2484a295a24de86fc/R/get_vcov.R#L105

**Comment by vincentarelbundock** [comment]
No, but you can just get the covariance you meant with the sandwich package and feed that matrix directly to the vcov argument
